grafana/overlays/emea/devconfus2021/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../../base
+  - opf-monitoring
```

Comment:
All these resources can go into base. Also there's no need to have a specific subfolder for a single resource, just plain single base dir with all resources on the same level is enough.

Reply:
Resolved in latest commit.